bundles/org.openhab.binding.mystrom/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -157,6 +158,14 @@
 		<state pattern="%.3f %unit%" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="ws-channel">
+		<item-type>Number:Power</item-type>
```

Comment:
I'm a bit confused by the name Ws - are we dealing with Watt seconds? In that case it's Energy, not Power.